client/src/app/shared/components/simple-task-viewer/simple-document-viewer.tsx

```diff
@@ -128,47 +146,50 @@ export const SimpleDocumentViewer = <FetchType,>({
         </div>
       }
       customControls={[
-        <div
-          className={css(
-            editorStyles.codeEditorTab,
-            "language-toggle-group-container"
-          )}
-          key="code-language-toggle"
-        >
-          <ToggleGroup
-            aria-label="code content type selection"
-            className="language-toggle-group"
+        <>
+          {refreshControl}
```

Comment:
`customControls` is an array of elements, so this can just be its own array element.  No need to change the other controls.

The "code-language-toggle" should be last in the array.

Reply:
Updated - nice catch.